src/challenge/ChallengeContext.ts

```diff
@@ -283,6 +287,13 @@ class ChallengeContextClass {
       breakpoints: number[],
       mode: "debug" | "run" = "debug"
     ) => {
+      const addFiles: AdditionalFilesContents =
+        this.challenge.state.additionalFilesLoaded;
+
+      let additionalCode = "";
+      Object.keys(addFiles).forEach((filename) => {
+        additionalCode += `with open("${filename}", "w") as f:f.write(r"""${addFiles[filename]}""")\n`;
```

Comment:
converted to r""" to avoid files with \ causing havoc. I suppose in this templated setup a file with a triple quite can still cause issues, but perhaps that's unlikely to happen (?). Alternatively, we could encode/decode